### DISCUSSION TITLE
Skin swapper for the app

### DISCUSSION THREAD

**Discussion by Thinkseal** [modrinth/.github#145] [Feature requests]
Is a skin library something that the Modrinth app could see in the future? It's the one thing I think is so important, yet almost no third-party launcher has.

**Comment by sizquirt** [comment]
Most people use the official launcher which is mostly always better, the only launchers that DO have it suck.
This is just my opinion so don't take it harshly

**Reply by Thinkseal** [reply]
ye this is true unless you want to mod minecraft then you basically need a 3th party launcher.
